configurations/pq/pq_count_lat_lng.yaml

```diff
@@ -0,0 +1,151 @@
+#TODO:
```

Comment:
Is this some "bad merge" artefact? This seems to be a new file, but pq count products are in `pq_count_alber_{all|annual}.yaml`

Reply:
Just a "bad Damien" artefact. I didn't look closely enough to work out what was current and what wasn't.